### DISCUSSION TITLE
How to create/train LORA model in extension?

### DISCUSSION THREAD

**Discussion by iradeyal2** [d8ahazard/sd_dreambooth_extension#507] [General]
I can see how to load an existing LORA model in the pulldown on the left of the DREAMBOOTH tab, but I can't for the life of me figure out how to start the process and CREATE a new LORA model that I can then train. Is there a function I'm missing to get the process started? Has anyone posted a guide to using Lora in the extension? 

Excited to try this out...

THANKS!

**Comment by Thomas-MMJ** [comment]
need to start web ui with --test-lora and then in the advanced section of dreambooth you can check the lora box.

**Comment by iradeyal2** [comment]
That worked thanks!

**Comment by iradeyal2** [comment]
Sorry, still not sure how to get this working. I checked the box and created a model but no lora model is available in the lora pulldown on the far left.

![Web capture_12-12-2022_162653_127 0 0 1](https://user-images.githubusercontent.com/54294736/207196723-ad072f2e-9a2e-4f84-bdca-50a78dc72973.jpeg)

**Comment by iradeyal2** [comment]
I found this thread in issues which is helpful: https://github.com/d8ahazard/sd_dreambooth_extension/issues/486
